Luna_Streaming_2.8.3_Release_Notes.md

```diff
@@ -60,32 +60,53 @@ To fix this, you can manually log into the ZooKeeper, BookKeeper, and Function W
+This is because there is a breaking API change in Apache Pulsar 2.8.0 (and therefore in Luna Streaming 2.8.0) related to the `SchemaInfo` java class.
 More context [here](https://github.com/apache/pulsar/issues/11338).
 
-If you are upgrading from Luna Streaming Distribution 2.8.0 you don't have to do any particular action.
+If you are upgrading from Luna Streaming Distribution 2.8.0, you don't have to do any particular action.
 
 ### Packaging
 
-Luna Streaming Distribution comes with a few different packages that aim to different purposes. 
-The distributions are available as Docker images and tarballs, both of them follow the same packaging patterns.
+Luna Streaming Distribution comes with different packages tooled for different purposes. 
+The distributions are available as Docker images and tarballs, and both methods follow the same packaging patterns.
 
 #### Patterns
 
-* **lunastreaming**: the basic Luna Streaming Distribution, **including Pulsar SQL** feature.
-* **lunastreaming-core**: the basic Luna Streaming Distribution, **without Pulsar SQL** feature. You should pick this one if you are not interested in using Pulsar SQL features.
-* **lunastreaming-all**: it contains the Core Luna Streaming Distribution, including **Pulsar Offloaders** and the **Datastax Pulsar IO Connectors** listed above. You should pick this if you are interested in using the Datastax connectors or the offloading feature.
+* **lunastreaming**: the basic Luna Streaming Distribution **including Pulsar SQL** feature.
+* **lunastreaming-core**: the basic Luna Streaming Distribution **without Pulsar SQL** feature. Pick this distribution if you don't want Pulsar SQL features.
+* **lunastreaming-all**: contains the Core Luna Streaming Distribution, including **Pulsar Offloaders** and the **Datastax Pulsar IO Connectors** listed above. Pick this distribution if you want the Datastax connectors or the offloading feature.
 
-# Releases
+#### Builtin connectors
+* cassandra-enhanced-pulsar-sink-1.6.1-nar.nar
+* pulsar-cassandra-source-1.0.3.nar
+* pulsar-io-data-generator-2.8.3.1.0.5.nar
+* pulsar-io-debezium-mongodb-2.8.3.1.0.5.nar
+* pulsar-io-debezium-mssql-2.8.3.1.0.5.nar
+* pulsar-io-debezium-mysql-2.8.3.1.0.5.nar
+* pulsar-io-debezium-oracle-2.8.3.1.0.5.nar
+* pulsar-io-debezium-postgres-2.8.3.1.0.5.nar
+* pulsar-io-elastic-search-2.8.3.1.0.5.nar
+* pulsar-io-jdbc-clickhouse-2.8.3.1.0.5.nar
+* pulsar-io-jdbc-mariadb-2.8.3.1.0.5.nar
+* pulsar-io-jdbc-postgres-2.8.3.1.0.5.nar
+* pulsar-io-jdbc-sqlite-2.8.3.1.0.5.nar
+* pulsar-io-kafka-2.8.3.1.0.5.nar
+* pulsar-io-kinesis-2.8.3.1.0.5.nar
+* pulsar-snowflake-connector-0.1.7.nar
+* pulsar-protocol-handler-kafka-2.8.0.1.0.16.nar
+* starlight-rabbitmq-2.8.0.1.1.27-ls-1.nar
+* pulsar-kafka-proxy-2.8.0.1.0.16.nar
+
+## Releases
 
-## Luna Streaming Distribution 2.8.3 1.0.5
```

Comment:
This part (also the connectors) part is generated automatically, can we leave it as is?